www-apps/webdavcgi/webdavcgi-1.1.2.ebuild

```diff
@@ -55,6 +56,16 @@ REQUIRED_USE="|| ( mysql postgres sqlite )"
 
 CGIBINDIR="cgi-bin"
 
+PATCHES=(
+	# Fix unescaped braces, which adresses #674772 and #658470
+	#
+	# The patch originates from
+	# https://github.com/DanRohde/webdavcgi/commit/04e79b7ecbaf3aae5ab813cd4fc0a009c72b1580
+	# and can be remove as soon as this ebuild gets bumped to 1.1.3 which
+	# already includes the change.
+	"${FILESDIR}/${P}-fix-unescaped-braces.patch"
+)
+
 src_compile() {
 	if use suid; then
 		WEBDAVWRAPPERS="webdavwrapper" # Standard UID/GID wrapper
```

Comment:
Is the export definitely needed?

Reply:
`WEBDAVWRAPPERS` is used within `src_install()` later on.